src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -8047,4 +8047,16 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_InterceptsLocationAttributeUnsupportedSignature_Title" xml:space="preserve">
     <value>'InterceptsLocationAttribute(string, int, int)' is not supported. Move to 'InterceptableLocation'-based generation of these attributes instead. (https://github.com/dotnet/roslyn/issues/72133)</value>
   </data>
+  <data name="IDS_FeatureUserDefinedCompoundAssignmentOperators" xml:space="preserve">
+    <value>user-defined compound assignment operators</value>
+  </data>
+  <data name="ERR_OperatorsMustBePublic" xml:space="preserve">
+    <value>User-defined operator '{0}' must be declared public</value>
+  </data>
+  <data name="ERR_BadIncrementOpArgs" xml:space="preserve">
+    <value>Overloaded instance increment operator '{0}' takes no parameters</value>
```

Comment:
Consider using the imperative form here: _must_ take no parameters, for example. Right now, it's unclear from the wording how many parameters are supposed to be present.